### DISCUSSION TITLE
Fileupload in repeater

### DISCUSSION THREAD

**Discussion by benkomatej** [OFFLINE-GmbH/oc-boxes-support#134] [Help]
Hello guys .. please is possible use fileupload in repater ? Thank you.

**Comment by tobias-kuendig** [comment]
@benkomatej we have (undocumented) experimental support for this. Simply name your repeater field `repeater_items` and every `image/images` and `file/files` relation will be saved.

See https://github.com/OFFLINE-GmbH/oc-boxes-support/issues/126#issuecomment-2107018892 and also  https://github.com/OFFLINE-GmbH/oc-boxes-free-plugin/blob/main/models/RepeaterItem.php for the implementation.
